### ISSUE TITLE
Icon request: Italian Air Force

### ISSUE THREAD

**Issue by ingroxd** [krisu5/aegis-icons#92]
- URL of the site or app **(required):** [Italian Air Force](http://www.aeronautica.difesa.it/Pagine/default.aspx)
- Logo resources for the icon **(optional, but recommended):** [Logo](https://en.wikipedia.org/wiki/Italian_Air_Force#/media/File:Coat_of_arms_of_the_Italian_Air_Force.svg)

Also, I think Italia Militare could be the name for the icon, either.

**Comment by krisu5** [comment]
Okay, but it's not gonna be easy to see on Aegis.

**Comment by krisu5** [comment]
I tried for a moment, but in the end I feel like this is monumental amount of work for one icon. Specially for icon that's for such niche subject. I just don't want to spend many hours on simplifying emblem for small scale and still be unsatisfied by the result. I'm sorry if this is disappointing, but vector editing isn't as straight forward as photoshopping and I'm not that skilled either.

I made generic "plane" icon though:
https://github.com/krisu5/aegis-icons/blob/master/PNG/Generic/Plane.png
